### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by PublicHealthDataGeek** [nmfs-opensci/quarto-thesis#10] [General]
Hello
I have a really simple question although it is stupid.  I presume to get quarto-thesis onto my laptop I download the zip file via the code button.  But then I dont understand what I do to use it in R. I would normally create a project and have the files in then.  How does the installing the extension fit into it?  Is that something I need to run in the terminal in the folder that I have downloaded? 
Many thanks for any advice you can give for a newbie!
Best wishes

Caroline

**Comment by eeholmes** [comment]
Hi, I made some videos to show you how to open your unzipped folder in RStudio and get started:

* Show me how to download a zip and open in RStudio: [Video](https://youtu.be/33l8GhtUfnU)
* Show me how to use this repo as a template and then clone to my computer with RStudio: [Video](https://youtu.be/smzNQtogSaI)

**Reply by PublicHealthDataGeek** [reply]
Oh my goodness!  Thank you so much for that.  I would have figured it out eventually but you have probably saved me half a day!  Many thanks.
